src/platform/imx8/lib/dai.c

```diff
@@ -19,6 +19,18 @@ static struct dai esai[] = {
 	.index = 0,
 	.plat_data = {
 		.base = ESAI_BASE,
+		.fifo[SOF_IPC_STREAM_PLAYBACK] = {
+			.offset		= ESAI_BASE + REG_ESAI_ETDR,
+			.depth		= 384,  /* in bytes */
```

Comment:
for consistency either put them in words or change SAI too please :)

Reply:
Well the EDMA code is common. Should I put them in words before your PR is merged and have it be equally broken with the SAI? I can wait out on the merge of #1969 and only then change it. (also from our discussion I think we could set it to 192, however only 96 words are available; what should I set it to once that PR is merged?)

Reply:
if you choose bytes I can change my PR 
no problem

Reply:
I will try on a separate branch to rebase on top of your PR. Will report if the value of 96 (which is the number of words) works.

Reply:
Ah, the value 96 does work just fine. So, should I put the "96" value even before your PR is merged (and in the meanwhile have it broken for both of us) or should I wait for the merge and then update on my end? (and in case this goes in before you'd change it for the ESAI)

Reply:
I dunno what is the correct way, we could try to have my PR merged so this one goes in 
correct I think that would be the simplest thing to do

Reply:
Yeah I believe that would be the proper way to do it. The other way would have been to merge the ESAI first and have you correct the value in your own PR which seems slightly more awkward in my opinion.

Reply:
Alright, yours got merged first, updated on my side.